app/components/collectionShow/reducer.tsx

```diff
@@ -52,14 +54,26 @@ export function reducer(
         failedToLoadingCollection: true,
       };
     }
-
+    case ACTION_TYPES.COLLECTION_SHOW_START_TO_GET_PAPERS: {
+      return {
+        ...state,
+        isLoadingPaperToCollection: true,
+      };
+    }
+    case ACTION_TYPES.COLLECTION_SHOW_FAILED_TO_GET_PAPERS: {
+      return {
+        ...state,
+        isLoadingPaperToCollection: false,
+      };
+    }
     case ACTION_TYPES.COLLECTION_SHOW_SUCCEEDED_GET_PAPERS: {
       return {
         ...state,
+        isLoadingPaperToCollection: false,
         paperIds: action.payload.paperIds,
         sortType: action.payload.sort as AUTHOR_PAPER_LIST_SORT_TYPES,
-        papersTotalPage: action.payload.totalPages,
-        papersCurrentPage: action.payload.number + 1,
+        totalPaperListPage: action.payload.totalPages,
+        currentPaperListPage: action.payload.page + 1,
```

Comment:
You should do these data change in API module.
Because API is the only exception for our page rule.

- Component, Actions, Reducers, Helpers, ... -> Use Scinapse page number rule. (Non- side effect data layer)
- API Module -> Raw data layer (side effect data layer) - should return data that Scinapse rule is applied.